sql/2020/11_Security/cryptominer_usage.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+SELECT
+  month,
+  client,
+  freq
+FROM (
+  SELECT
+    LEFT(_TABLE_SUFFIX, 10) AS month,
+    IF(ENDS_WITH(_TABLE_SUFFIX, '_desktop'), 'desktop', 'mobile') AS client,
+    COUNT(0) AS freq
+  FROM
+    `httparchive.technologies.*`
+  WHERE
+    category = 'Cryptominers'
+    OR category='Cryptominer'
+  GROUP BY
+    _TABLE_SUFFIX)
+ORDER BY
+  client,
+  month,
+  freq DESC
```

Comment:
I still don't see need of inner loop. This gives same result:

```sql
#standardSQL
SELECT
  LEFT(_TABLE_SUFFIX, 10) AS month,
  IF(ENDS_WITH(_TABLE_SUFFIX, '_desktop'), 'desktop', 'mobile') AS client,
  COUNT(0) AS freq
FROM
  `httparchive.technologies.*`
WHERE
  category = 'Cryptominers'
  OR category='Cryptominer'
GROUP BY
  _TABLE_SUFFIX
ORDER BY
  client,
  month,
  freq DESC
```

Reply:
Indeed, those seem exactly the same. Updated.